src/Microsoft.DocAsCode.Plugins/BuildPhase.cs

```diff
@@ -5,9 +5,7 @@ namespace Microsoft.DocAsCode.Plugins
 {
     public enum BuildPhase
     {
-        PreBuild,
-        Build,
+        PreBuildBuild,
```

Comment:
better naming?

---
In reply to: [89570173](https://github.com/dotnet/docfx/pull/923#discussion_r89570173) [](ancestors = 89570173,89570051)